website/content/api-docs/jobs.mdx

```diff
@@ -2682,3 +2697,93 @@ WebSocket Request: /v1/job/actions-demo/action?action=weather&allocID=8614ed07-4
 # Toronto: ⛅️  -1°C
 {"stdout":{"data":"VG9yb250bzog4puF77iPICAtMcKwQw0K"}}
 ```
+
+## Create Job Version Tag
+
+This endpoint creates a tag for a job version.
```

Comment:
I think it's less necessary here than in the CLI section, but we might also consider expanding on why a tag is useful here. Something like "Applying a tag to a version prevents it from being garbage-collected, and makes it accessible by name when comparing versions"

Reply:
done - added a partial and included